Include/httpClient/httpProvider.h

```diff
@@ -65,6 +65,7 @@ STDAPI HCHttpCallSetContext(
     _In_ void* context
     ) HC_NOEXCEPT;
 
+
 /////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Extra line intentional?

Reply:
yep.  i love me some extra lines before the //// 

---
In reply to: [181256500](https://github.com/Microsoft/libHttpClient/pull/128#discussion_r181256500) [](ancestors = 181256500)